### ISSUE TITLE
Save Form ends up with an error

### ISSUE THREAD

**Issue by kappalab** [yuzoolcode/yuzoolthemes-tracking#34]
Save Form in JustForms became ending up with the error in the attached screen shot. It used to work fine until about an hour ago. The form has a lot of elements; is there any limitation in the number of elements in a form? 
![saveformerror](https://user-images.githubusercontent.com/50013541/60739886-7cdcb900-9f9e-11e9-8c52-2bd8167227a8.png)

**Comment by yuzoolcode** [comment]
No there isn't a limit @kappalab but longer forms may take longer to save

Are you still having the same issues since the weekend? 
Thanks

**Comment by kappalab** [comment]
Yes, I still have the error. It never worked throughout this weekend :(

**Comment by kappalab** [comment]
I quit the web browser and restarted macOS and the issue was gone. I did not want to quit my web browser because I already added quite a few elements unsaved when the error started. Maybe a web browser issue but I am not sure. I hope the issue won't come back and anyway, I will save more often.

**Comment by kappalab** [comment]
Now, the error came back, and I believe I hit some kind of limitation in the number of elements (fields) or data size because if I add a field, the error shows up but if I delete the filed, then it saves properly. So, I can intentionally bring up the error now. To determine what goes wrong, please let me know what information you require.

**Comment by yuzoolcode** [comment]
Hmm maybe you're right and the form is too large?
We'll add more memory capabilities soon and let me know if it keeps happening in this situation thanks

**Comment by kappalab** [comment]
Thank you very much and at this moment,there is no change. Please let me know once you have added more memory.

**Comment by yuzoolcode** [comment]
That is something we will do but not immediately. Can I suggest you prune the form in the meantime?

**Comment by kappalab** [comment]
I will wait for you to implement it because I am still about a half way to complete the form and will need to add twice (or more) numbers of fields.
